tools/SpectrePch.hpp

```diff
@@ -4,26 +4,220 @@
 #ifndef SPECTRE_PCH_HPP
 #define SPECTRE_PCH_HPP
 
-// Include STL headers
 #include <algorithm>
 #include <array>
+#include <bitset>
+#include <cassert>
+#include <cctype>
+#include <cerrno>
+#include <chrono>
+#include <climits>
+#include <cmath>
+#include <codecvt>
+#include <complex>
+#include <csignal>
 #include <cstddef>
+#include <cstdint>
+#include <cstdio>
+#include <cstdlib>
+#include <cstring>
+#include <ctime>
+#include <deque>
+#include <exception>
+#include <execinfo.h>
+#include <filesystem>
+#include <forward_list>
+#include <fstream>
+#include <functional>
+#include <initializer_list>
+#include <iomanip>
+#include <ios>
+#include <iosfwd>
+#include <istream>
+#include <iterator>
 #include <limits>
+#include <list>
+#include <locale>
+#include <map>
 #include <memory>
+#include <mutex>
+#include <new>
+#include <numeric>
+#include <optional>
 #include <ostream>
+#include <queue>
+#include <random>
+#include <regex>
+#include <set>
+#include <sstream>
+#include <stack>
+#include <stdexcept>
+#include <string>
+#include <thread>
+#include <tuple>
+#include <typeinfo>
+#include <type_traits>
+#include <unordered_map>
+#include <unordered_set>
 #include <utility>
+#include <variant>
 #include <vector>
 
-#include <Utilities/ErrorHandling/Assert.hpp>
+#include <glob.h>
+#include <libgen.h>
+#include <unistd.h>
+
+// Cannot include catch
+// error: no member named 'Session' in namespace 'Catch'
+// #include <catch.hpp>
+
+#include <hdf5.h>
+
+#include <libxsmm.h>
+
+// Mac CI build couldn't find this
+// #include <sharp_cxx.h>
+
+#include <yaml-cpp/yaml.h>
+
+#include <charm++.h>
+#include <charm.h>
+#include <ckarrayindex.h>
+#include <converse.h>
+#include <pup.h>
+#include <pup_stl.h>
+
+// Python bindings are not always built
+// #include <pybind11/numpy.h>
+// #include <pybind11/operators.h>
+// #include <pybind11/pybind11.h>
+// #include <pybind11/stl.h>
+
+#include <gsl/gsl_errno.h>
+#include <gsl/gsl_fit.h>
+#include <gsl/gsl_integration.h>
+#include <gsl/gsl_matrix_double.h>
+#include <gsl/gsl_multifit.h>
+#include <gsl/gsl_multiroots.h>
+#include <gsl/gsl_poly.h>
+#include <gsl/gsl_sf_bessel.h>
+#include <gsl/gsl_spline.h>
+#include <gsl/gsl_vector_double.h>
+
+#include <brigand/adapted/fusion.hpp>
+#include <brigand/adapted/variant.hpp>
+#include <brigand/brigand.hpp>
+#include <brigand/sequences/back.hpp>
```

Comment:
I think we can remove this from `src/Evolution/Systems/GrMhd/GhValenciaDivClean/Subcell/NeighborPackagedData.hpp` instead of including it here.